cfgmgr/intfmgr.cpp

```diff
@@ -539,12 +539,16 @@ bool IntfMgr::doIntfAddrTask(const vector<string>& keys,
 
         std::vector<FieldValueTuple> fvVector;
         FieldValueTuple f("family", ip_prefix.isV4() ? IPV4_NAME : IPV6_NAME);
-        FieldValueTuple s("scope", "global");
-        fvVector.push_back(s);
-        fvVector.push_back(f);
 
-        m_appIntfTableProducer.set(appKey, fvVector);
-        m_stateIntfTable.hset(keys[0] + state_db_key_delimiter + keys[1], "state", "ok");
+        // Don't send link local config to AppDB and Orchagent
+        if (ip_prefix.getIp().getAddrScope() != IpAddress::AddrScope::LINK_SCOPE)
```

Comment:
Do you have to check this in delete case as well?

Reply:
I don't think we have to check since delete case would be fine even if the key does not exist in appDB.  To keep it consistent, I added that too.